apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -855,6 +799,28 @@ export class ApiModelGenerator {
     }
   }
 
+  /**
+   * @param nodesToCapture - A list of child nodes whose token ranges we want to capture
+   */
+  private _buildExcerptTokens(astDeclaration: AstDeclaration,
+    nodesToCapture: IExcerptBuilderNodeToCapture[]): IExcerptToken[] {
+
+    const excerptTokens: IExcerptToken[] = [];
+
+    // Build the main declaration
+    ExcerptBuilder.addDeclaration(excerptTokens, astDeclaration, nodesToCapture, this._referenceGenerator);
```

Comment:
Should this be built into the ExcerptBuilder.addDeclaration call?

Reply:
I did that originally, but this seemed simpler because (1) it avoids exposing the `Collector` to `ExcerptBuilder` and (2) it avoids having to pass `this._collector` and `this._referenceGenerator` every time `ApiModelGenerator` generates excerpts.